server/ocsp.go

```diff
@@ -435,7 +435,7 @@ func (srv *Server) NewOCSPMonitor(config *tlsConfigKind) (*tls.Config, *OCSPMoni
 
 		// Check whether need to verify staples from a client connection depending on the type.
```

Comment:
This comment is confusing.   Feels like throughout we are introducing confusion (at least I get confused!) by mixing the c-word "client" as a mix of TLS directional role and NATS client.   For OCSP and TLS code recommend trying to disambiguate and lean towards TLS directional role for clarity.

Should it read: 

// Check whether need to verify **server's** staple from **outbound** connection depending on the type.

Reply:
Updated the terminology to use `peers` instead of server/client since cannot make the distinction at this stage yet.